source/Mortgage.cpp

```diff
@@ -46,8 +46,8 @@ int64_t Mortgage::Maximum(int64_t annualRevenue, int creditScore, double current
 
 
 // Create a new mortgage of the given amount.
-Mortgage::Mortgage(int64_t principal, int creditScore, int term)
-	: type(creditScore <= 0 ? "Fine" : "Mortgage"),
+Mortgage::Mortgage(const string &type, int64_t principal, int creditScore, int term)
+	: type(std::move(type)),
```

Comment:
If you're going to use `std::move` the parameter should be a `std::string`, `std::string &`, or `std::string &&`. You can't move from a `const std::string &`.